Update 3.rendering.md (#5049)
nuxt/framework: docs/content/2.guide/1.concepts/3.rendering.md

### Examples

> Client-side rendering is a good choice for heavily interactive **web applications** that don't need indexing or ones frequently used by the same users. It can leverage browser caching to skip the download phase on subsequent visits, such as **SaaS, back-office applications, or Online Games**.

Reviewer: ```suggestion
Client-side rendering is a good choice for heavily interactive **web applications** that don't need indexing or whose users visit frequently. It can leverage browser caching to skip the download phase on subsequent visits, such as **SaaS, back-office applications, or online games**.
```
